examples/CXX_host/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.11)
+
+project(CXX_host LANGUAGES CXX)
+
```

Comment:
You should set:
```
set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
set(CMAKE_CXX_EXTENSIONS OFF)
```
as these flags are not inherited from the `XCFun::xcfun` target.